src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -159,10 +159,18 @@ def get(self, request, organization):
                 else:
                     queryset = queryset.none()
 
+        is_detailed = request.GET.get("detailed", "0") != "0"
```

Comment:
I see that we do this in a couple other places, but I'm kind of confused why we explicitly need `"0"` here and why we can't just check for if `is_detailed = request.GET.get("detailed") == "1"` 

I was looking at https://github.com/getsentry/sentry/pull/13925#discussion_r301767480 and I think we needed to explicitly set `"0"` there for it to be backward compatible ? (if I read that correctly)

Reply:
@MeredithAnya I think we use any positive integer as True and `0` as False, and for this one I want to set if False by default. Ex: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_teams.py#L95

Reply:
yeah I was looking at this example as well:
https://github.com/getsentry/sentry/blob/79a3d43f866419bd228cd81f9e26735ab62b8c5b/src/sentry/api/endpoints/organization_processingissues.py#L23